src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -44,23 +44,24 @@ class AppDeployIntegrationTest
     deploymentIds.length should be(1)
     val deploymentId = deploymentIds.head
 
-    val apiPostEvent = waitForEvent("api_post_event")
+    log.info("waiting for deployment success")
+    val events: Map[String, Seq[CallbackEvent]] = waitForEvents(
+      "api_post_event", "group_change_success", "deployment_info",
+      "status_update_event", "status_update_event",
+      "deployment_success")(30.seconds)
```

Comment:
It is optional in the technical sense as well. you could use empty parenthesis. Of course, you could also move the timeout parameter list before the events but that's not better in my opinion. If it is better in your opinion, I can change that.

Or do you suggest something else?